atoms/build.rs

```diff
@@ -22,7 +22,7 @@ fn main() {
             }
         }
     }
-    include!("../style/counter_style/predefined.rs");
+    include!("./predefined_counter_styles.rs");
```

Comment:
A better way to do this would probably be to have the import script duplicate this file into the atoms directory.

Reply:
I did think about that but this directory doesn't exist at import time either (at least not when calling git filter-repo) because it's not upstream. I suppose we could add a post rebase step to sync and squash this file? But that seems like it might be quite fragile?